### ISSUE TITLE
Possible support for BWTA, or other terrain analyzers?

### ISSUE THREAD

**Issue by SavantAI** [Lamarth/BWAPI-CLI#3]
Hello,

Thanks for making this, I can finally program a bot in C# due to this.

I've seen the original BWAPI-CLI, and it has support for BWTA, but it runs on an old version of BWAPI (3.7.3)

I was wondering if you could add support to this version for BWTA?

Thanks in advance!

**Comment by Lamarth** [comment]
I don't think it can be connected through two different versions of BWAPI at once. I'm not really sure - I'm not much of a C++ programmer, more of a F#/C# programmer who had a burst of motivation. If you have an idea that's feasible, I *may* be able to find some time for it.
